frontend/src/components/common/Input/Input.tsx

```diff
@@ -0,0 +1,34 @@
+import { forwardRef, InputHTMLAttributes } from 'react';
+
+import { css } from 'styled-components';
+
+import * as S from '@/components/common/Input/Input.styles';
+import { InputStatus } from '@/components/common/Input/Input.type';
+
+interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
+  width?: string;
+  height?: string;
+  status?: InputStatus;
+  message?: string;
+  id: string;
+  label: string;
+  $css?: ReturnType<typeof css>;
+}
+const Input = forwardRef<HTMLInputElement, InputProps>(
+  (
+    { width = '80%', height = '4.8rem', status = 'default', message, id, label, ...props }: InputProps,
+    ref,
+  ): JSX.Element => {
```

Comment:
`JSX.Element` 반환 타입이 없어도 충분히 타입스크립트에서 타입을 추론 가능할 것 같습니다.
```suggestion
  ({ width = '80%', height = '4.8rem', status = 'default', message, id, label, ...props }: InputProps, ref) => {
```